src/darwin/Framework/CHIP/MTRBaseSubscriptionCallback.h

```diff
@@ -119,10 +119,12 @@ class MTRBaseSubscriptionCallback : public chip::app::ClusterStateCache::Callbac
 
     // OnEventData and OnAttributeData must be implemented by subclasses.
     void OnEventData(const chip::app::EventHeader & aEventHeader, chip::TLV::TLVReader * apData,
-        const chip::app::StatusIB * apStatus) override = 0;
+        const chip::app::StatusIB * apStatus) override
+        = 0;
```

Comment:
Not a fan of the style here, but it is what it is.

Reply:
Yeah, must be new clang-format logic... I've had a quick look but didn't spot a setting to keep these on one line.